docs/tutorials/noise.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>is reffered to as the --&gt; is referred to as the</p><p>a simplifying assumption you can make -&gt; a simple assumption you can make</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/quantum/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>